process/block/preprocess/sovereignChainTransactions.go

```diff
@@ -214,3 +220,89 @@ func (sct *sovereignChainTransactions) isTransactionEligibleForExecution(tx *tra
 func isCriticalError(err error) bool {
 	return err != nil && !errors.Is(err, process.ErrHigherNonceInTransaction)
 }
+
+// RequestTransactionsForMiniBlock requests missing transactions for the given mini block
+func (sct *sovereignChainTransactions) RequestTransactionsForMiniBlock(miniBlock *block.MiniBlock) int {
+	if miniBlock == nil {
+		return 0
+	}
+
+	//TODO: Implement functionality for sovereign chain
+	return 0
+}
+
+// RequestBlockTransactions requests missing transactions from the given body
+func (sct *sovereignChainTransactions) RequestBlockTransactions(body *block.Body) int {
+	if check.IfNil(body) {
+		return 0
+	}
+
+	//TODO: Implement functionality for sovereign chain
+
```

Comment:
I do not think this funciton has to be reimplemented. we can use the same for base.

Reply:
Right, I've deleted it so the base one can be used